spec/features/feedback_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+RSpec.feature 'Customer feedback', js: true do
+  around do |example|
```

Comment:
This isn't actually doing anything as you need to call `example.run`. Also may as well just inline it into the single `scenario` where it's used.